windows/jitsi-meet/micro.js

```diff
@@ -0,0 +1,124 @@
+const {ipcRenderer} = require('electron');
+
+/**
+ * The RTCPeerConnection to main window.
+ */
+let peerConnectionMicro;
+
+/**
+ * The remote video from main window.
+ */
+let largeVideo = document.createElement('video');
```

Comment:
I think you can move this to the HTML now, just query the selector in JS and do the needful there.